mathesar/pagination.py

```diff
@@ -67,7 +67,8 @@ def paginate_queryset(self, queryset, request, table_id,
                 filters=filters, order_by=order_by
             )
             # Convert the tuple keys into strings so it can be converted to JSON
-            group_count = {','.join(k): v for k, v in group_count.items()}
+            group_count = [{"columns": list(cols), "count": count}
```

Comment:
@eito-fis On second thought, I think changing this key to `values` instead of `columns` makes more sense, these aren't column names.